addons/mrp_operations/mrp_operations_view.xml

```diff
@@ -66,7 +66,7 @@
             <header>
                 <button name="button_start_working" string="Start" states="draft" class="oe_highlight"/>
                 <button name="button_resume" string="Resume" states="pause" class="oe_highlight"/>
-                <button name="button_done" string="Finished" states="startworking" class="oe_highlight"/>
+                <button name="action_button_done" string="Finished" states="startworking" class="oe_highlight" type="object"/>
```

Comment:
it will break workflow. button_done is signal name of workflow using this signal workflow and workflow transition triggered..